### PR TITLE
Update Meld, add 2 new contracts 

### PR DESCRIPTION
**Pull request by clawoo**

### CODE REVIEW

File: dApps/Meld.json [hunk 1]
```diff
@@ -150,15 +171,26 @@
             ]
         },
         {
-            "id": "v1TRHt",
+            "id": "lXQeog",             
             "purpose": "SPEND",
             "versions": [
                 {
                     "version": 1,
-                    "scriptHash": "ba3501cd170c96349c342c5ef4242c596b58afaecdcffc0bb04af0ec",
```
Comment (matiwinnetou) [inline dApps/Meld.json:158]: Why is this removed?

Comment (clawoo) [reply dApps/Meld.json:158]: Not removed, moved up so that the 2 scripts are in order of release on the network (staking 1 and 2)
